sources/packages/backend/libs/services/src/notifications/notification/notification-actions.service.ts

```diff
@@ -187,6 +188,42 @@ export class NotificationActionsService {
     );
   }
 
+  /**
+   * Creates a notification when an Application Offering Change Request is completed by the ministry.
+   * @param notification input parameters to generate the notification.
+   * @param auditUserId user that should be considered the one that is causing the changes.
+   * @param entityManager optional entity manager to execute in transaction.
+   */
+  async saveApplicationOfferingChangeRequestCompletedByMinistry(
+    notification: ApplicationOfferingChangeRequestCompletedByMinistryNotification,
+    auditUserId: number,
+    entityManager?: EntityManager,
```

Comment:
In this method `entityManager` should be mandatory.

Reply:
I agree. I also see that all the methods in this class have entityManager mandatory but it is not utilized in an optional way.